### PR TITLE
[FIX] web: use color-contrast instead of color-yiq

### PR DESCRIPTION
**Pull request by sdegueldre**
Previously, we were using the bootstrap 4 function "color-yiq" to choose
a text color depending on the background color. Since the migration to
BS5, color-yiq is no longer valid, and color-contrast should be used
instead, but the wowl-views migration was maintained in parallel from
the BS5 migration and some instance of this function call were added
back with it. This commit fixes those.

Functionally, this fixes the text color in the m2m_tags field.

### CODE REVIEW

File: addons/web/static/src/legacy/scss/utils.scss [hunk 1]
```diff
@@ -70,7 +70,7 @@
 //
 // Given two colors, returns the one which has the most constrast with another
 // given color. Careful: if you want to find the text color which will suit the
-// most on a given background color, you should use the 'color-yiq' function.
+// most on a given background color, you should use the 'color-contrast' function.
 //
 @function o-get-most-contrast($color, $c1, $c2, $background: #FFFFFF, $threshold: false, $cross-mix: true) {
```
Comment (sdegueldre) [inline addons/web/static/src/legacy/scss/utils.scss:75]: We can probably replace this function with the BS one but this is out of scope for this PR

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

File: addons/web/static/src/legacy/scss/utils.scss (hunk 1 above)
Comment (pparidans) [reply addons/web/static/src/legacy/scss/utils.scss:75]: https://github.com/odoo/enterprise/pull/29802
